src/steps/users/converter.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  createIntegrationEntity,
+  Entity,
+} from '@jupiterone/integration-sdk-core';
+
+import { Entities } from '../constants';
+import { HexnodeUser } from '../../types';
+
+export function createUserEntity(user: HexnodeUser): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: user,
+      assign: {
+        _type: Entities.USER._type,
+        _class: Entities.USER._class,
+        _key: `${user.name}-${user.id.toString()}`,
```

Comment:
Can we stick to the convention you've been using such as 

```suggestion
        _key: `hexnode_user:${user.name}-${user.id.toString()}`,
```

I'm also curious, is there a reason why you think `user.id` won't suffice? If there is, this is super fine, but if `id` is enough then we could just use that too.

Reply:
Oh yea about that I originally was just using `${user.id.toString()}` as the `_key` but it errors out since there's a minimum character requirement that's why I added the user. I forgot we append `${Entity._type}:...` to the `_key` my bad.